lib/pipelines/rtsp-jpeg-pipeline.js

```diff
@@ -0,0 +1,20 @@
+const RtspPipeline = require('./rtsp-pipeline')
+
+const JPEGDepay = require('../components/jpegdepay')
+
+class RtspJpegPipeline extends RtspPipeline {
+  /**
+   * Create a pipeline which is a linked list of components.
+   * Works naturally with only a single component.
+   * @param {Array} components The ordered components of the pipeline
+   */
+  constructor (rtspConfig) {
+    super(rtspConfig)
+
+    const jpegDepay = new JPEGDepay()
+
+    this.append(jpegDepay)
+  }
+}
+
+module.exports = RtspJpegPipeline
```

Comment:
With standard-js files should end in newline, might be worth to add a standard --fix hook somewhere?

Reply:
The tests will catch that if it's not the case. I prefer that one runs `standard --fix` themselves if needed, I don't like automatic changes when committing.

Reply:
You're right, it just didn't show up when viewing the file online on github

Reply:
newline !== empty line

Some editors/viewers will show a final empty line when you have a newline character at the end, some don't (like github's viewer)